src/schema.sql

```diff
@@ -36,30 +52,28 @@ CREATE TABLE bounds (
 CREATE INDEX bounds_gist ON bounds USING GIST(geom);
 CREATE INDEX bounds_idx ON bounds(name);
 
-DROP TABLE IF EXISTS users;
-CREATE TABLE users (
-    id          BIGSERIAL PRIMARY KEY,
-    access      TEXT,
-    username    TEXT UNIQUE NOT NULL,
-    password    TEXT NOT NULL,
-    email       TEXT UNIQUE NOT NULL,
-    meta        JSONB
-);
-
-DROP TABLE IF EXISTS users_tokens;
 CREATE TABLE users_tokens (
     name        TEXT,
-    uid         BIGINT,
+    uid         BIGINT REFERENCES users(id),
```

Comment:
Changes seems unrelated to the 'history retrieval schema' I think we should move it to another PR.

Reply:
Removed in https://github.com/mapbox/Hecate/pull/206/commits/7493e0efafaeec0af7b70cbab56df3f1b16525d5 -- while these are changes we should make, i'm removing them from the current PR to reduce scope for what will already be a large change.